lib/utils/validate.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable newline-per-chained-call */
+/**
+ *
+ * @description This function validate input field that need an alphabet
+ * @param {object} req
+ * @param {string} name
+ * @returns {*} *
+ */
+function validateName(req, name) {
+  req
+    .check(name)
+    .notEmpty()
+    .withMessage(`${name} is required`)
+    .isAlpha()
+    .trim()
+    .withMessage(`${name} must be alphabets`);
+}
+/**
+ *
+ * @description This function validate input field that need an alphabet
+ * @param {object} req
+ * @param {string} email
+ * @returns {*} *
+ */
+function validateEmail(req, email) {
```

Comment:
You don't need to pass in `email` as an argument to the function since the function only validates email.